### ISSUE TITLE
Break out latex dissertation template into seperate repo

### ISSUE THREAD

**Issue by dmadisetti** [bibekanandadatta/LaTeX-Templates#1]
How come Adyota is the only one you whatsapp?

Also add `.DS_store` into your `.gitignore`

**Comment by bibekanandadatta** [comment]
Ahh! Managing multiple repos. Alright; I will do it. Here you go: https://github.com/bibekananda-datta/JHU-Dissertation-Template.
The latest versions are getting approved on Overleaf as well. I can share link there if you use that.

You can also text me as well ha ha!

**Comment by dmadisetti** [comment]
Cool! Yeah, I already saw it on overleaf. There's a way to connect a GitHub repo with overleaf that you might want to look at

**Comment by bibekanandadatta** [comment]
Initially I just dumped all the templates in a single repo so wasn't convenient for me connect to Overleaf's multiple project. Now that I separated out, I already thought of doing it. Thanks! 

Current version on Overleaf is not same as the new repo. Overleaf takes a day or two have the updated template after their review process. On Yoda's feedback, I heavily updated the template until yesterday.
